src/glib/base/gix.h

```diff
@@ -236,7 +247,7 @@ class TGixItemSet {
 
     /// Ask child vectors about their memory usage
     uint64 GetChildMemUsed() const {
-        return ChildrenData.GetMemUsed(true);
+        return TMemUtils::GetExtraMemberSize(ChildrenData);
```

Comment:
Depends on what we want to measure. Here I measured only the space that the children occupy